app/utils/transformers/DepartureTransportMeansTransformer.scala

```diff
@@ -34,7 +35,7 @@ class DepartureTransportMeansTransformer @Inject() (implicit ec: ExecutionContex
             userAnswers =>
               val dtmIndex: Index = Index(i)
               val pipeline: UserAnswers => Future[UserAnswers] =
-                set(VehicleIdentificationTypePage(dtmIndex), typeOfIdentification) andThen
+                set(TransportMeansIdentificationPage(dtmIndex), TransportMeansIdentification(typeOfIdentification, "")) andThen
```

Comment:
Let's do the reference data call once here. Then eventually we can remove all reference data lookups from the CYA page.

Reply:
There is a brief example of this in the DocumentTransformer although this is still a WIP (and untested)